docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 worker: &worker
-  image: addons/olympia
+  build: .
```

Comment:
As a side note: This is only for development, once this PR is r+ the image will default to addons/olympia again.